buildstockbatch/workflow_generator/residential_hpxml.py

```diff
@@ -430,9 +434,27 @@ def create_osw(self, sim_id, building_id, upgrade_idx):
 
         if "include_annual_bills" in util_bills_rep_args_avail:
             util_bills_rep_args["include_annual_bills"] = True
```

Comment:
Maybe we should now default this to False? buildstockbatch only ever got the runner-registered values -- so since we now have `register_annual_bills`, we may as well not write the `results_bills.csv` file, right?

Reply:
I think it's useful to write for inspection by a user. Much easier to read than the results.json.